benchmark-service/benchmark/benchmark.py

```diff
@@ -200,8 +207,37 @@ async def compare(request, userdata):  # pylint: disable=unused-argument
     return await render_template('benchmark', request, userdata, 'compare.html', context)
 
 
+async def query_github(app):
+    global START_POINT
+    github_client = app['github_client']
+    request_string = f'/repos/hail-is/hail/commits?since={START_POINT}'
```

Comment:
So was the decision to not include the sha because it doesn't work?

Reply:
I think a specific sha was working but sha=main didn't work